content/articles/how-to-load-and-query-a-csv-file-created-with-python-in-ibm-db2-database/index.md

```diff
@@ -0,0 +1,176 @@
+
+![go to ui](ibm-demo13.png)
+
+Once you open the `Go to UI` option, at the top left corner of your Db2 service dashboard click the menu bar icon, where you will see the option for `Data` as shown below.
+
+![democsvfile](ibm-demo14.png)
+
+You can either drag and drop the CSV file into the File selection space, or use the `Browse files` option in the File selection space to look for the CSV file in your local directories.
+
+![democsvfile](ibm-demo15.png)
+
+In the Selected file section at the right corner of your screen, the name of the CSV file you uploaded will appear. In this case the `demo.csv` file is what you can see here.
+
+![democsvfile](ibm-demo22.jpg)
+
+You'll be prompted to choose a Schema from a list of options, you can use the one available for you in the schema option for lite plan users.
+
+![democsvfile](ibm-demo22new.jpg)
+
+After which you can click on the `new table` option at the top right corner and input any name of your chioce for your table and click on `Next` which will take you to a table of your CSV file.
+Make sure your `Header in first row` option is marked. 
+
+![democsvfile](ibm-demo21.jpg)
+
+You can now click on `Next` which will take you to the last process in loading your CSV file.
+
+![democsvfile](ibm-demo19.jpg)
+
+Finally you can now begin loading your CSV file into your Db2 database. If no errors were identified during the process, you will receive a success message.
+
+![democsvfile](ibm-demo20.jpg)
+
+Now you have loaded your CSV file into Db2 database you can now run some SQL queries on it.
+
+### Running SQL queries with your CSV  data in your db2 database.
+To run your SQL queries in your Db2 database you will need to go to the `Run SQL` icon in the menu bar section at the top left corner of your screen.
+
+![democsvfile](ibm-demo24.jpg)
+
+Once you click on it, you will be taken to where you can either upload a file to query or create a new query page. In this case you already have your CSV file you will need to query in the database, so all you will need to do is to create a new query page to query your CSV dataset in the database.
+
+![democsvfile](ibm-demo25.jpg)
+
+Once you click on the `Create new` option, the SQL editor you will need to run your queries will open.
+
+![democsvfile](ibm-demo26.jpg)
+
+You can now run some queries on your data in the database. In this case you will run a query to check if your `DEMO` database you created is present in the database using the query code below.
+
+```sql
+SELECT * FROM DEMO;
+```
+  
+![democsvfile](ibm-demo27.jpg)
+
+To rap this up you will run a query to add more data into your database using the queries as shown below.
+```sql
+INSERT INTO DEMO VALUES('Jhon', 'Bobson', 'email@Bobson');
+INSERT INTO DEMO VALUES('Mary', 'Madeline', 'email@Bobson');
+INSERT INTO DEMO VALUES('Okwonu', 'Ese', 'email@Bobson');
+INSERT INTO DEMO VALUES('Queen', 'VIctor', 'email@Bobson');
+INSERT INTO DEMO VALUES('Ruona', 'Ighere', 'email@Bobson');
+```
```

Comment:
Why do we need to add more data?

In the CSV file that you are going to share, make sure you add more data there, and we may remove this code snippet, since it may cause unnecessary confusion to the reader.

Reply:
All comments have been noted and I will make adjustment as said..
As for that sql query to add more data into the database.
I included it because from the topic "How to load and query a..........." The query is also an important aspect of the topic so I decided to run a query to include more data into the database as suggested by the name of the tutorial.